docs/commands/clip.md

```diff
@@ -0,0 +1,30 @@
+# clip (Windows Only)
```

Comment:
The Nushell `clip` command works across Linux, macOS, and Windows.

Reply:
removed (Windows Only)